client/reader/stats.js

```diff
@@ -77,5 +79,9 @@ export function recordUnfollow( url ) {
 }
 
 export function recordTrack( eventName, eventProperties ) {
+	const subCount = SubscriptionStore.getTotalSubscriptions();
```

Comment:
Hmmmm. We probably need to keep around the total number of subs. Sounds like we'll need to mod #3418.